specs/subresourceintegrity/spec.markdown

```diff
@@ -263,9 +263,9 @@ list, and use that metadata to validate the resource (as described below in
 the "[parse metadata][parse]" and "[get the strongest metadata from
 set][get-the-strongest]" algorithms).
 
-When a hash function is determined to be insecure, user agents MUST deprecate
-and eventually remove support for integrity validation using that hash
-function.
+User agents MAY deprecate support (by blocking loads) for integrity validation using 
```

Comment:
Well, my main concern with fail open is that we shouldn't limit our options for a future new version of SRI, so this isn't quite that. But, yeah, you have already responded on that thread. I mostly think we have reached the point of diminishing returns on what we should do here, so I am happy with whatever the consensus is ;)